### DISCUSSION TITLE
Adam 1.3 & 1.3 queries

### DISCUSSION THREAD

**Discussion by JAF62** [eduardofilo/RG350_adam_image#167] [General]
1 - I am currently using 1.3 and I rebuilt my favourites and rom preferences because of the simple menu changes from 1.2
If I upgrade to 1.3.1, will I have to do this again due to the downgrading of Simple Menu compared to 1.3?

2 - HDMI. I have tried this today and it seems to work well other than menu layout and distorted / flickering Open Dingux loading screens.

But no sound. 

Are there plans to upgrade so that sound goes through HDMI to TV or other HDMI target?
Or is there a known way to adjust a config or settings file to achieve this?
Thanks in advance for any answer.
I'm generally really pleased with Adam and it has made using my RG350 so much more enjoyable 😊

**Comment by eduardofilo** [comment]
1 - The situation of rom_preferences in the different Adam versions is this:
    * Adam 1.2, 1.3.1: rom_preferences are stored all together at the `~/.simplemenu/rom_preferences` root level.
    * Adam 1.3: rom_preferences are stored in subdirectories according to the different systems.

So the way to convert your rom_preferences between 1.3 and 1.3.1 is mixing all the subdirectories content to the root level. That have the inconvenient of collides between ROMs with same name in different systems (the example I always give is 1942, which exists for a large number of system). It's a step backwards, I know, but the problems that people reported having with the version of SimpleMenu installed in 1.3 encouraged me to downgrading in 1.3.1. In fact, the one installed in 1.3 was not official, I took the risk of installing it precisely because of the advantages of organizing rom_preferences.

2 - Those features are part of the system that I don't develop, I just integrate it into the image. I don't remember reading that HDMI audio support is going to be attempted, sorry.

**Reply by eduardofilo** [reply]
I'm thinking, if you don't have problems with SimpleMenu installed on 1.3, you don't need to upgrade to 1.3.1. Aside from the fact that you would probably lose some of your rom_preferences settings, you would gain nothing by upgrading.

**Reply by JAF62** [reply]
Thank you very much for the explanation and responses.
I think I will keep 1.3 as it all seems to be running well, and I would guess thar when Simple Menu is upgraded officially it will keep to the changed format in 1.3. 👍
